Documentation/project-docs/contributing.md

```diff
@@ -26,6 +26,17 @@ Pull Requests
 * **DO** refer to any relevant issues, and include [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) that automatically close issues when the PR is merged.
 * **DO** tag any users that should know about and/or review the change.
 * **DO** ensure each commit successfully builds.  The entire PR must pass all tests in the Continuous Integration (CI) system before it'll be merged.
-* **DO** address PR feedback in an additional commit(s) rather than ammending the existing commits, and only rebase/squash them when necessary.  This makes it easier for reviewers to track changes. If necessary, squashing should be handled by the merger using the ["squash and merge"](https://github.com/blog/2141-squash-your-commits) feature, and should only be done by the contributor upon request.
+* **DO** address PR feedback in an additional commit(s) rather than amending the existing commits, and only rebase/squash them when necessary.  This makes it easier for reviewers to track changes.
+* **DO** assume that ["Squash and Merge"](https://github.com/blog/2141-squash-your-commits) will be used to merge your commit unless you request otherwise in the PR. Reasons may include (but not limited to)
```

Comment:
> Reasons may include

I don't think it's necessary to include these reasons here, for two main reasons:

1. This is a maintainers' policy, not a contributors' policy. Contributors do not need to understand the policy in order to be effective.
2. Contributors are more likely to disagree with the reasons for this policy.

For this section, the bullet point is sufficient if it ends with "...otherwise in the PR."